enterprise/views.py

```diff
@@ -943,6 +949,16 @@ def get(self, request, enterprise_uuid, course_id):
         will be either redirected to LMS dashboard for audit modes or
         redirected to ecommerce basket flow for payment of premium modes.
         """
+        # We need to override these constants here, because they are calcualted at load time and there's no
+        # Site context at load time.
+        LMS_DASHBOARD_URL = urljoin(
+            get_configuration_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
+            '/dashboard',
+        )
+        LMS_COURSEWARE_URL = urljoin(
+            get_configuration_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
+            '/courses/{course_id}/courseware',
+        )
```

Comment:
I think you can declare a variable to avoid calling `get_configuration_value` twice. @Squirrel18 

```suggestion
        root_url = get_configuration_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
        LMS_DASHBOARD_URL = urljoin(
            root_url,
            '/dashboard',
        )
        LMS_COURSEWARE_URL = urljoin(
            root_url,
            '/courses/{course_id}/courseware',
        )
```

Reply:
Nice catch. @Jacatove